app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  before_action :set_ticket, only: %i[show update]
+  before_action :set_participating_seller, only: %i[create]
+
+  # GET /tickets
+  def index
+    @tickets = Ticket.all
+
+    render json: @tickets
+  end
+
+  # GET /tickets/1
+  def show
+    render json: @ticket
+  end
+
+  # POST /tickets
+  def create
+    puts(params[:number_of_tickets])
+    savedCorrectly = true
+    createdTickets = []
+
+    (1..params[:number_of_tickets]).each do |_i|
+      attributes = {}
+      attributes[:ticket_id] = SecureRandom.alphanumeric(5).upcase.insert(4, '-')
+      attributes[:participating_seller] = @participating_seller
+
+      @ticket = Ticket.new(attributes)
```

Comment:
I think there is a way you can make this fail if it doesn't create with a bang!

Reply:
You're right! Though it wasn't on new, it was on `save` adding the bang now